src/test/java/org/carrot2/elasticsearch/ClusteringActionIT.java

```diff
@@ -189,33 +186,6 @@ public void testNonexistentAlgorithmId() throws IOException {
         }
     }
 
-    @Test
```

Comment:
This is actually a sensible test to make sure we get the exception we expect to receive.